src/kaleidoscope/addr.h

```diff
@@ -25,19 +25,19 @@
 // as the keyboard has fewer than 256 keys.
```

Comment:
I would expect that it would be possible to simply remove this header file entirely. It came from the Qukeys plugin, and I don't know of anything else that uses it – almost certainly there are no third-party plugins that do so, as it's only been accessible here for a few months.

Reply:
I thought so, too but as there is a small chance that someone might rely on it, I didn't dare to remove it.

Reply:
I would encourage @algernon and @obra to get rid of it right away if they adopt this PR, since it is attempting to serve the same purpose as the thing that should be used instead.

Reply:
I'll remove it as soon as A&O approve it.